src/app/components/button/button.component.scss

```diff
@@ -28,3 +20,17 @@
     padding: 3px 8px;
   }
 }
+
+.medium {
+  font-size: 15px;
+  line-height: 18px;
```

Comment:
- check  src/scss/mixins/_text.scss and see if we can arrange the fonts in this file.